packages/drag-drop/src/elements/draggable-list/components/DropIndicator.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, LiHTMLAttributes } from 'react';
+import { useDraggableListContext } from '../../../utils/useDraggableListContext';
+import { StyledDropIndicator } from '../../../styled';
+
+/**
+ * @extends LiHTMLAttributes<HTMLLIElement>
+ */
+export const DropIndicator = forwardRef<HTMLLIElement, LiHTMLAttributes<HTMLLIElement>>(
+  (props, ref) => {
+    const { isHorizontal } = useDraggableListContext();
+
+    return <StyledDropIndicator {...props} isHorizontal={isHorizontal} ref={ref} />;
```

Comment:
Great catch. The indicator will realistically hold focus during keyboard interaction, so an `aria-label` is appropriate. 

I'll need to include focus styling for it as well, which I believe [Pane](https://garden.zendesk.com/components/pane#vertical) is a good guide for?

Reply:
Nevermind, I asked design. Not really sure about the focus ring.

Reply:
From @steue: `DropIndicator` _shouldn't_ be gaining focus, actually. A `Draggable` that's grabbed (as an overlay, or otherwise) is intended to retain focus. Applying `aria-hidden="true"` might be preferable in this case I believe.

Reply:
Yeah, we definitely don't want this to be a focusable element. However, that does not negate the need for a screenreader to be able to work through a list of elements and have each one of them read out. I don't believe we want `aria-hidden` as this is meaningful to the UI.

Reply:
I added `useText` with a required `aria-label`.